src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,78 @@
+var semver = require("semver");
+
+var AppDispatcher = require("../AppDispatcher");
+var JSONPUtil = require("../helpers/JSONPUtil");
+var MesosEvents = require("../events/MesosEvents");
+
+var MesosActions = {
+  requestVersionInformation: function (host) {
+    JSONPUtil.request(`${host}/version`).then(
```

Comment:
The version endpoint was (will be) introduced with mesos 0.26 thus there is only this endpoint or nothing. :)
—
Orlando Hohmeier

> On 09 Dec 2015, at 14:01, Pierluigi Cau <notifications@github.com> wrote:
> 
> In src/js/actions/MesosActions.js <https://github.com/mesosphere/marathon-ui/pull/473#discussion_r47086874>:
> 
> > @@ -0,0 +1,78 @@
> > +var semver = require("semver");
> > +
> > +var AppDispatcher = require("../AppDispatcher");
> > +var JSONPUtil = require("../helpers/JSONPUtil");
> > +var MesosEvents = require("../events/MesosEvents");
> > +
> > +var MesosActions = {
> > +  requestVersionInformation: function (host) {
> > +    JSONPUtil.request(`${host}/version`).then(
> Shouldn't we add Mesos version checks similarly to what we do for other actions to ensure the right endpoint is hit?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mesosphere/marathon-ui/pull/473/files#r47086874>.
>